src/nbl/ext/ImGui/shaders/common.hlsl

```diff
@@ -1,56 +1,26 @@
-#ifdef __HLSL_VERSION
```

Comment:
if its meant to be a header, then add a header guard

Reply:
resolved